scripts/create_config_zip.sh

```diff
@@ -0,0 +1,63 @@
+#!/bin/bash
+
+# Create links for all property files and logback configuration files
```

Comment:
It looks like this script creates copies of files instead of links as the comment states.